src/Skia/Avalonia.Skia/Compatibility/SkiaCompat.SKImageFilter.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using SkiaSharp;
+
+namespace Avalonia.Skia;
+
+internal static partial class SkiaCompat
```

Comment:
All of this reflection code can be avoided, if we will be able to push some new APIs to the 2.88.8/2.89.0 backport version.

Specifically backport these two methods from 3.0 to 2.88 which should be pretty easy to do:
https://github.com/mono/SkiaSharp/blob/9bb8b235fa203adc9cbe456433c66ccbb1b52992/binding/SkiaSharp/SKImageFilter.cs#L43
https://github.com/mono/SkiaSharp/blob/9bb8b235fa203adc9cbe456433c66ccbb1b52992/binding/SkiaSharp/SKImageFilter.cs#L109

Reply:
Both `SKImageFilter.CreateBlur` and `SKImageFilter.CreateDropShadow` are static methods, I believe you can use function pointers instead to avoid the delegate overhead.

Reply:
Yeah, I was thinking to do so.
But after discussion, we likely to make SkiaSharp 3.0 compatibility exclusive to .NET 8 by using UnsafeAccessor anyway.

Reply:
But ideally, I want to think about backporting some of these methods to SkiaSharp 2.88.x, if possible.

Reply:
I think that will be something I would prefer. Adding newer APIs that wrap older ones so people can use that for 2.x and then upgrades are simpler. I also want to use all the new things in 3.0, such as the UnmanagedCallersOnly. Right now for iOS I have GCHandles and other weird things. I need to make sure all the new things work in WASM (as that where all the issues were before) but this should reduce the ugly code I have to deal with Xamarin.iOS